### ISSUE TITLE
[cloud-tensorflow-mnist]: Add logits and labels for softmax_cross_entropy_with_logits

### ISSUE THREAD

**Issue by gokul-uf** [googlecodelabs/feedback#320]
In section 8 of the tutorial, the call to softmax is
`cross_entropy = tf.nn.softmax_cross_entropy_with_logits(Ylogits, Y_)`

Latest versions of TF require that you explicitly mention which arguments are logits and labels.

So, it should be 
`cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits = Ylogits, labels = Y_)`

**Comment by thehans** [comment]
I just experienced the same issue.
